Makefile

```diff
@@ -76,24 +76,25 @@ ci-dependencies-start:
 	--name splunk-s3-tests-authservice-$$BUILD_NUMBER \
 	--volume=$$PWD/volumes/authservice:/cert:ro \
 	--rm -d storjlabs/authservice:dev run \
-		--listen-addr :8000 \
+		--listen-addr :20000 \
 		--cert-file /cert/authservice-cert.pem \
 		--key-file /cert/authservice-key.pem \
 		--allowed-satellites $$(docker exec splunk-s3-tests-sim-$$BUILD_NUMBER storj-sim network env SATELLITE_0_ID)@ \
 		--auth-token super-secret \
-		--endpoint http://gateway:7777 \
+		--endpoint http://gateway:20010 \
 		--kv-backend memory://
 
 	docker run \
 	--network splunk-s3-tests-network-$$BUILD_NUMBER --network-alias gateway \
 	--name splunk-s3-tests-gateway-$$BUILD_NUMBER \
 	--rm -d storjlabs/gateway-mt:dev run \
-		--server.address :7777 \
-		--auth.base-url http://authservice:8000 \
+		--server.address :20010 \
+		--auth.base-url http://authservice:20000 \
 		--auth.token super-secret \
 		--domain-name gateway \
 		--insecure-disable-tls \
-		--insecure-log-all
+		--insecure-log-all \
+		--s3compatibility.fully-compatible-listing
```

Comment:
one thing to note (if it hasn't been already!) is we'll want to add this to the CI for gateway-mt builds as well, otherwise the tests will fail there

Reply:
Good point! Thanks. Done: https://review.dev.storj.io/c/storj/gateway-mt/+/6937.